views/documentation_word.rhtml

```diff
@@ -135,63 +135,56 @@
       <div class="">
         <span class="subtitle"><a id="databases_data_storages_and_their_versions">Databases, data storages, and their versions</a></span>
         <ul class="mt--5">
-          <li>PostgreSQL 9.6</li>
-          <li>Redis 9.6</li>
-          <li>...</li>
+          <% Rails.configuration.database_configuration.each do |env, db_data| %>
```

Comment:
We don't need to display information about DB for each environment, in most cases, it's the same DB. Maybe we can somehow check the used database adapter?